CHANGELOG.md

```diff
@@ -5,6 +5,20 @@ The change log describes what is "Added", "Removed", "Changed" or "Fixed" betwee
 
 ## 2.0.0-rc5 (UNRELEASED)
 
+### Changed
```

Comment:
The changed part is already added for RC5 right under added (line 32 in your changed file)